# FormationData.msg

float64 position_x
float64 position_y
float64 position_z

float64 homeLat
float64 homeLon
float64 homeAlt
time gps_time

float64 leaderHeading
float64 leaderCommand_yaw
float64 leaderAirspeed